util/verible-format.sh

```diff
@@ -31,13 +31,31 @@ fi
 # overwriting of uncomitted changes
 git add -u
 
-# get all system verilog files and pipe through style formatter
-find . -type f -name "*.sv" -o -name "*.svh" |                           \
-    xargs -n 1 -P $NUM_PROCS verible-verilog-format                      \
-    $VERIBLE_ARGS
+# By default format only files in allow list
+MODE=${MODE:-allowlist}
 
+case $MODE in
+    allowlist)
+        FILES_TO_FORMAT=`cat util/verible-format-allowlist.vbl | sed '/^#/d'`
+        ;;
 
-echo "Usign verible-verilog-format version $VERIBLE_VERSION" > $REPORT_FILE
+    all)
+        # get all system verilog files and pipe through style formatter
+        FILES_TO_FORMAT=`find . -type f -name "*.sv" -o -name "*.svh"`
+        ;;
+
+    *)
+        echo "verible-format.sh: Unknown mode $MODE"
+        exit 1
+        ;;
+esac
+
+echo $FILES_TO_FORMAT | \
+      xargs -n 1 -P $NUM_PROCS verible-verilog-format                      \
```

Comment:
```suggestion
      xargs -n 1 -P $NUM_PROCS verible-verilog-format \
```

or align with the backslash above.

Reply:
Fixed.